sky/clouds/cloud.py

```diff
@@ -485,5 +486,25 @@ def query_status(cls, name: str, tag_filters: Dict[str, str],
         """
         raise NotImplementedError
 
+    @classmethod
+    def create_image_from_cluster(cls, name: str, tag_filters: Dict[str, str],
```

Comment:
API, just from reading the iface
- name -> image_name (or cluster_name)?
- tag_filters: Dict[str, str] -> seems like backend knowledge leaked to here?